docs/zh-CN/docs/sql-manual/sql-functions/string-functions/oct.md

```diff
@@ -0,0 +1,55 @@
+---
+{
+    "title": "OCT",
+    "language": "zh-CN"
+}
+---
+
+<!-- 
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+## oct
+### description
+#### Syntax
+
+`VARCHAR oct(BIGINT num)`
+
+如果输入参数是数字，则返回八进制值的字符串表示形式；
```

Comment:
Is there any possibility to support more input types in the future?

Reply:
yes. we can support input as string. will add support for this in next PR.